kmath-geometry/src/jvmTest/kotlin/scientifik/kmath/geometry/Utils.kt

```diff
@@ -0,0 +1,35 @@
+package scientifik.kmath.geometry
+
+import scientifik.kmath.dimensions.Dimension
+import kotlin.math.abs
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+fun grid(xRange: ClosedRange<Double>,
+         yRange: ClosedRange<Double>,
+         step: Double): List<Pair<Double, Double>> {
+    fun generateList(range: ClosedRange<Double>): List<Double> =
+            generateSequence(range.start) { previous ->
+                if (previous == Double.POSITIVE_INFINITY) return@generateSequence null
+                val next = previous + step
+                if (next > range.endInclusive) null else next
+            }.toList()
+
+    val xs = generateList(xRange)
+    val ys = generateList(yRange)
+
+    return xs.flatMap { x -> ys.map { y -> x to y } }
+}
+
+fun <D : Dimension> assertVectorEquals(expected: Vector<Double, D>, actual: Vector<Double, D>, eps: Double = 1e-6) {
```

Comment:
Can be moved to another file like `testUtils.kt`

Reply:
Other functions from this file also used for testing, so going to rename this file to `testUtils.kt`, and leave all functions in here. ok?